ccdproc/ccddata.py

```diff
@@ -29,8 +29,8 @@ class CCDData(NDDataArray):
 
     Parameters
     -----------
-    data : `~numpy.ndarray` or :class:`~astropy.nddata.NDData`
-        The actual data contained in this `~astropy.nddata.NDData` object.
+    data : `~numpy.ndarray` or :class:`~ccdproc.CCDData`
+        The actual data contained in this `~ccdproc..CCDData` object.
```

Comment:
`~ccdproc.CCDData` instead of `~ccdproc..CCDData` (only one dot) or is this intentional?